Assets/Scripts/Characters/Humans/Hero.cs

```diff
@@ -1281,6 +1286,37 @@ public void Update()
             }
         }
 
+        private void CheckForScrollingInput()
+        {
+            if (InputManager.Key(InputHuman.ReelIn))
+            {
+                reelForce = -1f;
+            }
+            else if (InputManager.Key(InputHuman.ReelOut))
+            {
+                reelForce = 1f;
+            }
+            else if (InputManager.KeyUp(InputHuman.ReelIn) || InputManager.KeyUp(InputHuman.ReelOut))
+            {
+                reelForce = 0f;
+            }
```

Comment:
I believe you should use `&&` here instead of `||`.
And remove `else` because if one of the two cases up there is true, those lines will never be reached.

Reply:
I don't think that anyone would release the reel in and reel out key binds at the same time which would be what is required to make that if statement true (unless KeyUp always returns true if not pressing the key). I'm not sure but I think it is only true once on the frame that the key is released (or not pressed anymore). 

I realize now that I can get rid of that entire elseif statement because the reelForce will "zero" out at the end of the statement chain if scroll inputs aren't being used.